TDDPractice/packages.config

```diff
@@ -1,7 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="AngleSharp" version="0.14.0" targetFramework="net461" requireReinstallation="true" />
+  <package id="Microsoft.Win32.Registry" version="4.7.0" targetFramework="net461" requireReinstallation="true" />
   <package id="MSTest.TestAdapter" version="2.1.2" targetFramework="net461" />
   <package id="MSTest.TestFramework" version="2.1.2" targetFramework="net461" />
   <package id="Selenium.Support" version="3.141.0" targetFramework="net461" />
   <package id="Selenium.WebDriver" version="3.141.0" targetFramework="net461" />
+  <package id="SharpZipLib" version="1.2.0" targetFramework="net461" />
```

Comment:
@andreea-qa why do we need all of these extra dependencies?

Reply:
@nadvolod not sure why it was added... I removed it

Reply:
@nadvolod I actually added it back because it comes with WebDriverManager and we need it to make it work